addon/-private/relay.js

```diff
@@ -9,21 +9,26 @@ export default EmberObject.extend({
   changeset: null,
   key: null,
 
+  init() {
+    this._changedKeys = [];
```

Comment:
You're right, I can see now that such condition is not likely to arise. A couple of points on `rollback`:
* While I definitely see and have use for it, is it something we want on relays? In the same vein we could add `snapshot`, `validate` etc. Thoughts?
* Couldn't per-relay rollback be implemented on the changeset like `this.changeset._rollbackRelay(this.key)`? That could also be the way for those other methods.